src/client/tweetAnalysis.spec.js

```diff
@@ -30,6 +30,37 @@ describe('#TweetAnalysis', () => {
       mostFrequentWords(tweets).should.deep.equal(exampleFrequentWords);
     });
 

```

Comment:
Nah let's keep them for now and I'll send an email to Fabio asking if he wants us to consider them as individual words or not :)